src/components/Dialogs/Confirm/Confirm.tsx

```diff
@@ -1,11 +0,0 @@
-import * as React from 'react';
```

Comment:
why are we removing this component?

Reply:
I removed this component because there is no use case for it and in the new design it is not there. 
Maybe we need to discuss this in the next sync meeting